generators/typescript/utils/core-utilities/form-data-utils/src/FormDataWrapper.ts

```diff
@@ -0,0 +1,67 @@
+import { RUNTIME } from "@fern-typescript/runtime";
```

Comment:
@armandobelardo for anything that uses `RUNTIME` we end up having to put inside the `fetcher` utils -- see `stream` as an example

Reply:
updated!